libr/anal/p/anal_x86_cs.c

```diff
@@ -1625,6 +1625,7 @@ static void anop(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len, csh
 		.bits = a->bits
 	};
 	int regsz = 4;
+	static RRegItem regs[2];
```

Comment:
`regs` must be a static variable because its address is saved in `src/dst` and `op` will be used by other functions.